ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -225,7 +225,9 @@ sub pod2html {
     local $_;
 
     my $globals = init_globals();
-    $globals = parse_command_line($globals);
+    my $opts = process_command_line;
+    $globals = process_options($globals, $opts);
```

Comment:
It's confusing to use the same reference that you're passing in _by reference_ and updating in the sub as the LHS of the assignment. I'd either drop the assignment and LHS `$globals` or do a deep copy and do an actual assignment  - in this case, my preference would be the former (drop the assignment) since it preserves the orignal global semantics the rest of the code assumes.